libpod/container_validate.go

```diff
@@ -70,7 +70,7 @@ func (c *Container) validate() error {
 
 	// Rootless has some requirements, compared to networks.
 	if rootless.IsRootless() {
-		if len(c.config.Networks) > 0 {
+		if !c.config.NetMode.IsSlirp4netns() {
```

Comment:
I'm still against merging with this here - there's probably something on the frontend that's erroneously detecting `slirp4netns:rootlesskit` as a CNI network and adding it.

Reply:
understand, and i'm open to ideas what/where else may need to be adjusted to not need this "enabling" snippet in container_validate.go.. 

i don't however quite understand what is wrong with this logic, why would the lenght of networks given ever be a non-empty array, networks shouldn't be able to be specified.. the fixing of the erroneous detection of that parameter seems like a separate bug that needs to be fixed/handled in a different PR, no ?

Reply:
@mheon sorry my mistake.  I had fixed the detection but forgot to switch the condition here.  Pushed a new version with `if len(c.config.Networks) > 0`.

I am currently experimenting running CNI plugins inside of the rootless environment.  So I cannot promise this line will stay around for long :-)

Reply:
Fair enough - I'd be extremely interested to see proper rootless CNI!

Reply:
it will still be limited.  We first need to create a network namespace managed by slirp, and from that network namespace create new ones using CNI.  I am still unsure how to handle its UX nicely